torch/csrc/jit/serialization/python_print.cpp

```diff
@@ -822,6 +822,15 @@ struct PythonPrintImpl {
         body_ << "):\n";
         printBody(graph->block());
       } break;
+      case prim::ModuleDictIndex: {
+        const auto dict = node->inputs().at(0);
+        const auto key = node->inputs().at(1);
+        const auto out = node->outputs().at(0);
+        assignValuesToTheirUniqueNames(out);
```

Comment:
Do we need to call assignValuesToTheirUniqueNames here ?

Reply:
Yes, there's no name for it yet.